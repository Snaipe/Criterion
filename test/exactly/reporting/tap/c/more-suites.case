[act]

% more-suites.c.bin --tap

[assert]

stdout
  -transformed-by replace_version
  equals -contents-of expected/more-suites.txt
