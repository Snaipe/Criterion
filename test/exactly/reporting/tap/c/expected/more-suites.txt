TAP version 13
1..3
# Criterion <VERSION>
#
# Running 1 tests from suite2
ok - suite2::test 
#
# Running 1 tests from suite1
ok - suite1::test 
#
# Running 1 tests from disabled
ok - disabled::test  # SKIP test was skipped
